File: python/shuffle_the_array.py
nums = [2, 5, 1, 3, 4, 7]
n = 3


def shuffle(nums, n):
  shuffle_arr = []

  for i in range(n):
    shuffle_arr.append(nums[i])
    shuffle_arr.append(nums[i + n])

  return shuffle_arr




print(shuffle(nums, n))

File: ruby/shuffle_the_array.rb
nums = [2, 5, 1, 3, 4, 7]
n = 3

def shuffle(nums, n)
  new_arr = []

  n.times do |i|
    new_arr.push(nums[i])
    new_arr.push(nums[i + n])
  end

  new_arr
end

p shuffle(nums, n)
